pkg/plugin/sdk/error.go

```diff
@@ -0,0 +1,27 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sdk
+
+import "github.com/conduitio/conduit/pkg/foundation/cerrors"
+
+var (
+	// ErrBackoffRetry can be returned by Source.Read to signal the SDK there is
+	// no record to fetch right now and it should try again later.
+	ErrBackoffRetry = cerrors.New("backoff retry")
+
+	// ErrUnimplemented is returned in functions of plugins that don't implement
+	// a certain method.
+	ErrUnimplemented = cerrors.New("action not implemented")
```

Comment:
There is, but we shouldn't use it, this package is supposed to be completely independent, it will be extracted into a separate repo after all. We are only using `cerrors` here because otherwise the linter would complain, this will be easy to change once we extract the repo, and I think it's the only dependency we have towards internal Conduit packages.

Reply:
Sounds reasonable to me, will resolve this.